src/inc/corinfo.h

```diff
@@ -2068,7 +2068,7 @@ struct CORINFO_Object
 struct CORINFO_String : public CORINFO_Object
 {
     unsigned                stringLen;
-    const wchar_t           chars[1];       // actually of variable size
+    wchar_t                 chars[1];       // actually of variable size
```

Comment:
If this is for VS's `warning C4090: 'function': different 'const' qualifiers`, then there is a false-positive in VS2015: https://github.com/cesanta/v7/pull/519#issuecomment-162254678 (that case is not available in [documented example](https://msdn.microsoft.com/en-us/library/k77bkb8d.aspx)), meaning this change might not be required.